cudf/dataframe.py

```diff
@@ -279,7 +282,21 @@ def assign(self, **kwargs):
 
     def head(self, n=5):
         """
-        Returns the first n rows as a new DataFrame
+        Return the first n rows.
```

Comment:
Why remove "as a new DataFrame"? I think this is an important detail: head is implemented using slicing, which returns a DataFrame.